motoman_gp225_support/package.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp225_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman GP225.</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman GP225 series of manipulators..
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>GP225 - Default</li>
```

Comment:
Similar to #607, the `xacro:macro` appears to model what *Table 5-1: Basic Specifications (YR-1-06VX225-A00)* labels as *Not-equipped*:

![image](https://github.com/user-attachments/assets/11d96ea1-f116-4484-adff-237db83bbe18)

Should we update this as well:

```suggestion
      <li>GP225 - Not-equipped</li>
```

?

Reply:
After further reflection, I don't think we should start doing equipped variant. The equipped variant has extra brackets and  cables running outside of the arm which is not included in the 3D model and I don't have the 3D models for that optional package. It is mainly for SpotWelding application which I don't think is used much among ROS users.